systemd/kibana.service

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=Kibana 4
+BindsTo=probemanager.service
+Wants=elasticsearch.service
+After=elasticsearch.service
+Wants=probemanager.service
+Before=probemanager.service
+Wants=probelogger.service
+Before=probelogger.service
```

Comment:
Feel free to investigate. I think this is the best way to specify the downstream services so it is clear that probemanager, probereader, probelogger, and probetransmogrifier all start once kibana has started.